Makefile

```diff
@@ -51,7 +58,11 @@ init:
 
 build:
```

Comment:
I'm leaning towards having them consolidated under the `build` target. That way we don't have to make any custom instructions in the readme for windows users.